modules/install-sno_additional-requirements-for-installing-on-a-single-node-on-aws.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-preparing-to-install-sno.adoc
+
+:_content-type: CONCEPT
+[id="additional-requirements-for-installing-on-a-single-node-on-aws_{context}"]
+= Additional requirements for installing on a single node on AWS
+
+The AWS documentation for installer-provisioned installation is written with a high availability cluster consisting of three control plane nodes. When referring to the AWS documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
+
+* The required machines for cluster installation in AWS documentation indicates a temporary bootstrap machine, three control plane machines, and at least two compute machines. You require only one AWS instance for the control plane node and no worker nodes.
```

Comment:
Hi @johnwilkins, `a temporary bootstrap machine` is still required by SNO.

Reply:
Fixed.